erpnext/hr/doctype/employee/employee.py

```diff
@@ -456,3 +395,68 @@ def has_user_permission_for_employee(user_name, employee_name):
 def get_employee_from_user(user):
 	employee_docname = frappe.db.get_value('Employee', filters={'user_id': user})
 	return employee_docname
+
+
+def get_employees_who_have_birthday_today(date_today):
```

Comment:
- add line `date_today = getdate(date_today)` to make sure date str is converted to date object or empty date is converted to today
- add default value to argument `date_today=None`